indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamAppenderatorConfig.java

```diff
@@ -57,11 +57,13 @@ public class SeekableStreamAppenderatorConfig implements AppenderatorConfig
 
   private final SeekableStreamIndexTaskTuningConfig tuningConfig;
   private final int maxColumnsToMerge;
+  private final boolean shouldReleaseLockOnHandoff;
```

Comment:
We probably don't need an extra field or method here, just access it via the tuning config.